x-pack/plugins/security_solution/public/timelines/components/timeline/properties/index.tsx

```diff
@@ -92,8 +100,31 @@ export const Properties = React.memo<Props>(
       setShowTimelineModal(true);
       // eslint-disable-next-line react-hooks/exhaustive-deps
     }, []);
+    const { navigateToApp } = useKibana().services.application;
+    const currentTimeline = useShallowEqualSelector((state) =>
```

Comment:
please use `useDeepEqualSelector` and destructure only props you need, otherwise, the callback will re-render every time any of the rest props of the timeline change